docs/reference/env_variables.rst

```diff
@@ -0,0 +1,170 @@
+        | A list of device indices or UUIDs that will be exposed to applications.
+      - ``0,GPU-DEADBEEFDEADBEEF``
+
+    * - | ``HIP_FORCE_QUEUE_PROFILING``
+        | Used to run the app as if it were run in rocprof. Forces command queue
+        | profiling on by default.
+      - ON, OFF
+
+Debug environment variables
+===========================
+
+The debugging environment variables in HIP is collected in the next table. For more information, check :ref:`debugging_with_hip`
+
+.. include:: ../how-to/debugging_env.rst
+
+Memory management related environment variables
+===============================================
+
+The memory management related environment variables in HIP is collected in the 
+next table.
+
+.. list-table::
+    :widths: 70,15,15
+    :header-rows: 1
+
+    * - Environment variable
+      - Variable type
+      - Default value
+
+    * - | ``HIP_HIDDEN_FREE_MEM``
+        | Reserve free mem reporting in Mb, 0 = Disable
+      - ``uint``
+      - 0
+
+    * - | ``HIP_HOST_COHERENT``
+        | Coherent memory in ``hipHostMalloc``
+      - ``uint``
+      - 0
+
+    * - | ``HIP_INITIAL_DM_SIZE``
+        | Set initial heap size for device malloc. The default value corresponds to 8 MiB
+      - ``size_t``
+      - 8388608
+
+    * - | ``HIP_MEM_POOL_SUPPORT``
+        | Enables memory pool support in HIP
+      - ``bool``
+      - ``false``
+
+    * - | ``HIP_MEM_POOL_USE_VM``
+        | Enables memory pool support in HIP
+      - ``bool``
+      - | ``true`` on Windows,
+        | ``false`` on other OS
+
+    * - | ``HIP_VMEM_MANAGE_SUPPORT``
+        | Virtual Memory Management Support
+      - ``bool``
+      - ``true``
+
+    * - | ``GPU_MAX_HEAP_SIZE``
+        | Set maximum size of the GPU heap to % of board memory
+      - ``uint``
+      - 100
+
+    * - | ``GPU_MAX_REMOTE_MEM_SIZE``
+        | Maximum size , in Ki that allows device memory substitution with system
+      - ``uint``
+      - 2
+
+    * - | ``GPU_NUM_MEM_DEPENDENCY``
+        | Number of memory objects for dependency tracking
+      - ``size_t``
+      - 256
+
+    * - | ``GPU_STREAMOPS_CP_WAIT``
+        | Force the stream wait memory operation to wait on CP.
+      - ``bool``
+      - ``false``
+
+    * - | ``HSA_LOCAL_MEMORY_ENABLE``
+        | Enable HSA device local memory usage
+      - ``bool``
+      - ``true``
+
+    * - | ``PAL_ALWAYS_RESIDENT``
+        | Force memory resources to become resident at allocation time
+      - ``bool``
+      - ``false``
+
+    * - | ``PAL_PREPINNED_MEMORY_SIZE``
+        | Size in KBytes of prepinned memory
+      - ``size_t``
+      - 64
+
+    * - | ``REMOTE_ALLOC``
+        | Use remote memory for the global heap allocation
+      - ``bool``
+      - ``false``
+
+HIP environment variables tips
+==============================
+
+Please feel free to open PR with your environment variable tips. We will check and add to this section.
+
+* The performance can be improved at Gromacs HIP backend, when the
```

Comment:
The formatting of this bullet looks odd in terms of indenting - and bolding. 

![image](https://github.com/ROCm/HIP/assets/164940351/4a5d1b24-578b-4f9f-98d5-b7b198b5a45e)

Reply:
Fixed.